.github/workflows/code-quality.yml

```diff
@@ -1,110 +1,23 @@
-name: Code quality checks
+name: code-quality
 
-# Run this workflow every time a new commit pushed to your repository
 on:
   push:
     branches:
-      - main
-      - stable/*
-    tags:
-      - '*'
+      - master
+    paths:
+      - '**.py'
+      - '**.yml'
```

Comment:
Is there a specific reason why this is also run if `.yml` files are changed?

Reply:
I guess it's to check if the workflow was changed but then it should just be the file. OZ, OK & objects-api do all have this as well.

open klant for example also has a comment that it should not need changing, do you know where this was copied from?

https://github.com/maykinmedia/open-klant/blame/master/.github/workflows/code-quality.yml

Reply:
Not sure where it came from, I see it in other repos like SDG and OIP as well. I guess it would be better if it was just this file indeed, but it doesn't matter too much imo